GitUI/CommandsDialogs/FormBrowse.Designer.cs

```diff
@@ -1796,5 +1806,6 @@ private void InitializeComponent()
         private RevisionDiff revisionDiff;
         private ToolStripContainer toolPanel;
         private RevisionGpgInfo revisionGpgInfo1;
+        private ToolStripMenuItem toolStripMenuItemUndoLastCommit;
```

Comment:
This do not follow the style of other items, toolStripMenuItemUndoLastCommit has a long use less prefix toolStripMenuItem
undoLastCommitMenuItem is more according to the style

Reply:
changed.